src/site/styles/layout.less

```diff
@@ -83,11 +83,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
```

Comment:
More specificity is better, it adds the distinction that these rules are not to be used on other components and are specific to a single component.